spyder_notebook/widgets/dom.py

```diff
@@ -18,10 +18,7 @@ class DOMWidget(WebView):
 
     def __init__(self, parent):
         super(DOMWidget, self).__init__(parent)
-        if WEBENGINE:
-            self.page = self.page()
-        else:
-            self.page = self.page().mainFrame()
```

Comment:
Ok, now I see what's the problem: I made the `page` method an attribute of this widget, and that break everything!! My bad, sorry ;-)

Then let's change this to

```python
if WEBENGINE:
    self.dom = self.page()
else:
    self.dom = self.page().mainFrame()
```